### PR TITLE
ReiBar Permissions & Floating APC Fixes

### PR DESCRIPTION
**Pull request by Rei1226**
Floating APC fixed
Bartender now has access to kitchen/botany
Chef now has access to botany
Botanists now have access to kitchen

### CODE REVIEW

**Comment by ComicIronic** [comment]
Conflicto.

**Comment by Rei1226** [comment]
Think it's the changelog, hold on

File: html/changelogs/Rei1226.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 author: Rei1226
 changes:
-  wip: Overall changes to the bar/kitchen area
+- wip: Overall changes to the bar/kitchen area
```
Comment (ComicIronic) [inline html/changelogs/Rei1226.yml]: The - shouldn't be there

**Comment by Rei1226** [comment]
jesus christ ok that should fix it thank you comic

**Comment by ComicIronic** [comment]
>11 commits to 8 line changes

Heredth don't got shit on this.

**Pull request merged**